pyspark/graphar_pyspark/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
By design, the GraphArSession is based on the SparkSession that is singleton too (even in Python). And having such a "session-object" simplify the code: otherwise you need to pass `SparkSession` into each class and store link to it in every method... Because SparkSession contains `JVMView` that is the backbone of the whole python code.

@Corwinpro do you have any suggestion how to resolve it? Or do you think that it is better to pass SparkSession into every method even with knowledge that `SparkSession` itself is singleton (you cannot have two different SparkSession objects)